osmedeus-server.py

```diff
@@ -1,39 +1,47 @@
+#!/usr/bin/env python3
+
 import os
 import json
 import logging
 import argparse
-from pathlib import Path
 from pprint import pprint
-import execute
-import utils
+from pathlib import Path
 
 from flask import Flask, request, send_from_directory
 from flask_cors import CORS
+from flask_restful import Api, Resource, reqparse
 from flask_jwt_extended import (
     JWTManager, jwt_required, create_access_token,
     get_jwt_identity
 )
-from flask_restful import Api, Resource, reqparse
-# really stuff
-from rest.decorators import local_only
-from rest.cmd import Cmd
-from rest.authentication import Authentication
-from rest.configuration import Configurations
-from rest.workspace import Workspace, Workspaces
-from rest.activities import Activities
-from rest.logs import Logs
-from rest.modules import Modules
-from rest.routines import Routines
-from rest.bash_render import BashRender
-from rest.wscdn import Wscdn
-from rest.save import Save
-
-BASE_DIR = Path(os.path.dirname(os.path.abspath(__file__)))
 
-# Flask config stuff
+from Osmedeus.core import execute
+from Osmedeus.core import utils
+from Osmedeus.core.rest.decorators import local_only
+from Osmedeus.core.rest.cmd import Cmd
+from Osmedeus.core.rest.authentication import Authentication
+from Osmedeus.core.rest.configuration import Configurations
+from Osmedeus.core.rest.workspace import Workspace, Workspaces
+from Osmedeus.core.rest.activities import Activities
+from Osmedeus.core.rest.logs import Logs
+from Osmedeus.core.rest.modules import Modules
+from Osmedeus.core.rest.routines import Routines
+from Osmedeus.core.rest.bash_render import BashRender
+from Osmedeus.core.rest.wscdn import Wscdn
+from Osmedeus.core.rest.save import Save
+from Osmedeus.resources import *
+
+OSMEDEUS_HOME = str(Path.home().joinpath('.osmedeus'))
+
+# TODO: I think we need get it from the config file
+STORAGES_DIR = OSMEDEUS_HOME + '/storages'
```

Comment:
I can not find another solution. 

If you want to use it as service mode, you can add a new `--storage` option for `osmedeus-server.py` script may be